### PR TITLE
fix(curriculum): typos in Nutrition Label section

### PR DESCRIPTION
**Pull request by Kernix13**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Step 25 Removed comma
Step 53 Capitalized `sodium`

**Note**: FYI, I only changed 2 files but the console showed `3 files changed`. I'm not sure if that is an issue or not.

nutrition-3-files

### CODE REVIEW

**Review by gikf — CHANGES_REQUESTED** [review]
There's some additional file here - `tream`

**Comment by Kernix13** [comment]
What is `tream`? I only opened and changed the two lesson files. However, I used the git command `git fetch upstream` but once again I wasn't on the `main` branch. I was in my branch `fix/rothkopainting-typos`. Could that have anything to do with it? Regardless, do I need to do something?

**Comment by gikf** [comment]
Check the _Files changed_ tab, I'm not sure what it is and how it made to the commit (although image shows it as a part of commit), but that file will need to be removed from PR. It can be done as an additional commit.

**Comment by Kernix13** [comment]
So if I did `git reset HEAD` that wouldn't do it? And then the command to reset the add but I can't remember that one. And then instead of doing `git add .`, could I add just add the changes with the file names like `step-25.md`? 

Do you even recognize that code as something from freeCodeCamp? Do you have a link on how to remove that via another commit because I do not know how to do that?

**Review by jdwilkin4 — APPROVED** [review]
LGTM! 👍

**Review by gikf — APPROVED** [review]

File: curriculum/challenges/english/14-responsive-web-design-22/learn-typography-by-building-a-nutrition-label/step-025.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ dashedName: step-25
 
 # --description--
 
-Now update your `h1` selector to be `header h1`, to specifically target your `h1` element within your new `header`.
+Now update your `h1` selector to be `header h1` to specifically target your `h1` element within your new `header`.
```
Comment (ShaunSHamilton) [inline curriculum/challenges/english/14-responsive-web-design-22/learn-typography-by-building-a-nutrition-label/step-025.md:10]: I am not sure why we removed this comma. This sentence consists of two clauses.

Comment (ojeytonwilliams) [reply curriculum/challenges/english/14-responsive-web-design-22/learn-typography-by-building-a-nutrition-label/step-025.md:10]: It's always dangerous when I pretend to know grammar, but I don't think you need to have a comma between the main and subordinate clauses.

Comment (gikf) [reply curriculum/challenges/english/14-responsive-web-design-22/learn-typography-by-building-a-nutrition-label/step-025.md:10]: Sorry, I should have waited with merging for more insights from those not scared of comma grammar.

Comment (ShaunSHamilton) [reply curriculum/challenges/english/14-responsive-web-design-22/learn-typography-by-building-a-nutrition-label/step-025.md:10]: It is not a big deal; the sentence is relatively easy to follow. So, my nit-pickiness aside, this is fine.

**Pull request merged**